.circleci/config.yml

```diff
@@ -73,17 +73,19 @@ jobs:
           path: ~/test_results
       - run: bash <(curl -s https://codecov.io/bash) -t ${CODECOV_TOKEN}
       # collect docs
-      - run: mkdir -p ~/docs
       - run:
           name: Collect docs
           command: |
+            mkdir -p ~/docs
             for subdir in `find . -maxdepth 1 -mindepth 1 -type d`; do
               if [ -d $subdir/target/dokka ]; then
                 cp -r $subdir/target/dokka ~/docs/$subdir
               fi
             done
+            mkdir -p ~/docarchive
+            cd ~/docs && zip -r ~/docarchive/docs.zip .
```

Comment:
👀 , should we do something similar in arm?

Reply:
if there are really slow `store_artifact` steps, then i think so! https://support.circleci.com/hc/en-us/articles/360024275534-Uploading-artifacts-is-slow -- see also https://discuss.circleci.com/t/store-artifacts-step-makes-browser-sluggish-consume-cpu/38211 which I filed with them